packages/react-components/src/accordion/accordion.tsx

```diff
@@ -24,7 +24,7 @@ const StyledTrigger = styled(AccordionPrimitive.Trigger, {
   [`[data-state=open] > & ${StyledChevron}`]: {
     transform: 'rotate(180deg)'
   },
```

Comment:
@tmbalagan  can we also set the cursor on accordion trigger to `'not-allowed'` when accordion item is disabled ?